packages/esm-patient-test-results-app/src/overview/external-overview.component.tsx

```diff
@@ -1,7 +1,7 @@
-import React, { useCallback, useMemo } from 'react';
+import React, { useCallback } from 'react';
 import { EmptyState, ExternalOverviewProps, PanelFilterProps } from '@openmrs/esm-patient-common-lib';
 import DataTableSkeleton from 'carbon-components-react/es/components/DataTableSkeleton';
-import { parseSingleEntry, OverviewPanelEntry } from './useOverviewData';
+import useOverviewData, { parseSingleEntry, OverviewPanelEntry } from './useOverviewData';
```

Comment:
Why use a `default` export here? Just name all the exports.

Right now we (exclusively?) use default exports for components to directly support lazy loading.

Reply:
(I suggest that we correct this in another PR - right now the function / default export already existed)